src/webview/newProjectPanel.mts

```diff
@@ -1771,7 +1811,7 @@ export class NewProjectPanel {
                         </div>
                       </div>`
                       : `<h3 class="text-xl font-semibold text-gray-900 dark:text-white mb-4">
-                            Warning: Project Import Wizard may not work for all projects, and will often require manual correction after the import
+                            Warning: Project Import Wizard may not work for all projects, and will often require manual correction after the import. Swift Projects aren't supported at the moment.
```

Comment:
Maybe just say only C/C++ projects are supported, rather than swift isn't supported, as the import wizard also doesn't support Python?
```suggestion
                            Warning: Project Import Wizard may not work for all projects, and will often require manual correction after the import. Only C/C++ Projects are supported at the moment.
```